tests/e2e/specs/modules/analytics/write-scope-requests.test.js

```diff
@@ -111,22 +114,22 @@ describe( 'Analytics write scope requests', () => {
 
 	beforeEach( async () => {
 		scope = 'https://www.googleapis.com/auth/analytics.provision';
-		bypassCreatePropertyRequest = true;
-		bypassCreateProfileRequest = true;
+		interceptCreatePropertyRequest = false;
+		interceptCreateProfileRequest = false;
 
-		await activatePlugin( 'e2e-tests-proxy-auth-plugin' );
 		await activatePlugin( 'e2e-tests-site-verification-plugin' );
```

Comment:
The `e2e-tests-site-verification-plugin` plugin shouldn't be necessary now that you're using `setupSiteKit()` as it will set the site verification for you 👍 
```suggestion
```